great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3841,3 +3849,15 @@ def save_config_variable(
 
         with open(config_variables_filepath, "w") as config_variables_file:
             yaml.dump(config_variables, config_variables_file)
+
+    def _load_zep_config(self) -> GxConfig:
+        if self.root_directory:
+            zep_config_yaml = pathlib.Path(self.root_directory) / "zep_config.yaml"
```

Comment:
Going with a dumb method of identifying a zep config initially.
May change this before the merge or as a followup.